app/models/miq_provision_workflow.rb

```diff
@@ -15,11 +15,17 @@ def self.base_model
   # @return [Constant] A scoped provider constant name.
   #
   def self.class_for_platform(platform)
-    classy = find_matching_constant("ManageIQ::Providers::#{platform}") ? platform : platform.classify
+    vendor = find_matching_constant("ManageIQ::Providers::#{platform}") ? platform : platform.classify
 
-    find_matching_constant("MiqProvision#{classy}Workflow") ||
```

Comment:
NOTE I don't think this "ever" worked because e.g. `MiqProvisionCloudWorkflow` is an alias, so `"MiqProvisionCloudWorkflow".safe_constantize&.name == "MiqProvisionCloudWorkflow"` is never true.
`"MiqProvisionCloudWorkflow".safe_constantize => ManageIQ::Providers::CloudManager::ProvisionWorkflow`